### DISCUSSION TITLE
FTB essentials Server crash

### DISCUSSION THREAD

**Discussion by SsHro** [FTBTeam/FTB-Mods-Issues#366] [Q&A] [answered]
I don't really know why this is happening, any thoughts 
[crash-2022-07-16_11.04.47-server.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/9126228/crash-2022-07-16_11.04.47-server.txt)
?

**Comment by MichaelHillcox** [comment, marked as answer]
You're using the wrong mod version for the forge version you're attempting to use. Find the 1.18 version of essentials on curseforge
